core/src/datayoga_core/resources/schemas/connections/postgresql.schema.json

```diff
@@ -9,7 +9,12 @@
     },
     "database": { "description": "DB name", "type": "string" },
     "user": { "description": "DB user", "type": "string" },
-    "password": { "description": "DB password", "type": "string" }
+    "password": { "description": "DB password", "type": "string" },
+    "connect_args": {
+      "description": "Additional DB connection arguments",
+      "type": "object",
+      "additionalProperties": true
```

Comment:
here we can specify all the properties explicitly rather than additionalProperties true, no?

Reply:
It's hard to specify all of them. Because each db have own params.

Reply:
yes, makes sense.